utils/get_supported_scylla_base_versions.py

```diff
@@ -18,6 +18,7 @@
 supported_src_oss = {'2021.1': '4.3', '2022.1': '5.0', '2022.2': '5.1', '2023.1': '5.2', '2023.2': '5.3'}
 # If new support distro shared repo with others, we need to assign the start support versions. eg: centos8
 start_support_versions = {'centos-8': {'scylla': '4.1', 'enterprise': '2021.1'}}
+start_support_cloud_provider = {'azure': {'scylla': '5.2', 'enterprise': '2023.1'}}
```

Comment:
>in that case, we will fail somewhere else, most likely when running init_and_verify_sct_config()

you just introduced the start_support_cloud_provider variable and used in only in set_start_support_version. to "Setting start support version". how it can affect init_and_verify_sct_config()?

> perhaps, we won't need it anymore, once 5.2 will be old enough (and not supported anymore)..

we just released 5.2. when it will be not supported anymore?



anyway, we can leave it as is because we are not introducing new cloud support every week.